src/modules/shared/layouts/AppLayout/AppLayout.tsx

```diff
@@ -125,6 +129,26 @@ const AppLayout: FC = ({ children }) => {
 		[dispatch]
 	);
 
+	const getVisitorSpaces = useCallback(async (): Promise<Visit[]> => {
+		if (!user || [GroupName.KIOSK_VISITOR, GroupName.ANONYMOUS].includes(user.groupName)) {
+			setVisitorSpaces([]);
+			return [];
+		}
+		const { items: spaces } = await VisitsService.getAll({
+			page: 1,
+			size: 10,
+			orderProp: 'endAt',
+			orderDirection: OrderDirection.asc,
```

Comment:
You want descending here.
Higher dates => low dates
recent dates => to older dates
```suggestion
			orderDirection: OrderDirection.desc,
```


https://github.com/viaacode/hetarchief-client/assets/1710840/e3bb1696-d5b4-4017-86e1-071cd59a39bd